### DISCUSSION TITLE
Official Research Paper Yolov8

### DISCUSSION THREAD

**Discussion by JSNN170** [ultralytics/ultralytics#8066] [Q&A]
Hi !

Is there an official research paper released demonstrating the performances of Yolov8 ?

Thank you.

**Comment by glenn-jocher** [comment]
Hi there!

As of now, there isn't an official research paper published for YOLOv8. The model is built on continuous improvements and community feedback, focusing on practical applications and performance enhancements rather than formal academic publication. For the most detailed and up-to-date information on YOLOv8, including its features and capabilities, please refer to the [official documentation](https://docs.ultralytics.com/).

Thanks for your interest! 😊
